src/Util.cc

```diff
@@ -435,6 +444,23 @@ ignition::gazebo::Entity topLevelModel(const Entity &_entity,
   return modelEntity;
 }
 
+//////////////////////////////////////////////////
+std::string topicFromScopedName(const Entity &_entity,
+    const EntityComponentManager &_ecm, bool _excludeWorld)
+{
+  std::string topic = scopedName(_entity, _ecm, "/", true);
+
+  if (_excludeWorld)
+  {
+    // Exclude the world name. If the entity is a world, then return an
+    // empty string.
+    topic = _ecm.Component<components::World>(_entity) ? "" :
+      topic = removeParentScope(removeParentScope(topic, "/"), "/");
```

Comment:
I think you can remove the `topic =` on the second line here

Reply:
Fixed in f0ced9aae58ce985e877d73b44cad1e9a6c0d9a8